templates/user/register.html.twig

```diff
@@ -21,6 +21,10 @@
                 {% for flash_error in app.flashes('verify_email_error') %}
                     <div class="alert alert__danger">{{ flash_error }}</div>
                 {% endfor %}
+                {% if mbin_sso_registrations_enabled() %}
+                    <h4 class="text-muted" style="text-align: center; margin:0 !important;">{{ 'Register using' }}</h4>
```

Comment:
To use the translation system you can change it to
```suggestion
                    <h4 class="text-muted" style="text-align: center; margin:0 !important;">{{ 'register_using_sso'|trans }}</h4>
```
Then you have to create an entry in `messages.en.yml` with the key `register_using_sso`